README.md

```diff
@@ -763,6 +780,6 @@ Jan Delgado (jdelgado at gmx dot net)
 
 ## Copyright and license
 
-Copyright (c) 2017-2019 Jan Delgado.
+Copyright (c) 2017-2020 Jan Delgado.
 rabtap is licensed under the GPLv3 license.
```

Comment:
If you are interested, here is the script I wrote to install on OS X:
```bash
curl -SL https://github.com/jandelgado/rabtap/releases/download/v1.23/rabtap-v1.23-darwin-amd64.zip | tar xvz -C . 
rm README.md
mv bin/rabtap-darwin-amd64 /usr/local/bin/rabtap
rmdir bin # only deletes non empty folders
chmod +x /usr/local/bin/rabtap
```
Ideally,  it would be on http://brew.sh/